packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,143 @@
+
+* `system.client.concurrency_rate_limit.violation`: This event is fired when the framework is in Enable mode. This event is fired when a specific client, IP address, or device identifier combination makes more than two concurrent requests. The System Log contains information about the client ID, IP address, device identifier, and the actual user if the user already has a valid session.
+
+* `system.client.rate_limit.violation.preview `: This event is fired when the framework is in Preview mode. This event is fired when a specific client, IP address, or device identifier combination exceeds the total limit of 60 requests per minute. However, the end user wouldn't see a rate-limit violation. The System Log contains information about the client ID, IP address, device identifier, and the actual user if the user already has a valid session.
+
+* `system.client.concurrency_rate_limit.violation.preview`: This event is fired when the framework is turned on in Preview mode. This event is fired when a specific client, IP address, Device token combination makes more than two concurrent requests. However, the end user wouldn't see a rate-limit violation. The System Log contains information about the client ID, IP address, Device identifier, and the actual user if the user already has a valid session.
+
+### Check your rate limits with Okta Rate Limit headers
+
+The Rate Limit headers returned when client-based rate limiting is enabled are very similar to the headers returned today through the [org-wide rate limits](/docs/reference/rl-best-practices/). The difference is that the header value is specific to a given client/IP/device combination rather than the org-wide Rate Limit values. Okta provides three headers in each response to report client-specific rate limits.
+
+> **Note:** If client-based rate limiting is in Preview or Disable mode, headers that are returned still reflect the org-wide rate limits.
+
+For client-specific rate limits, three headers show the limit that is being enforced, when it resets, and how close you are to hitting the limit:
+
+`X-Rate-Limit-Limit` - The rate limit ceiling that is applicable for the current request to the specific client/IP/device identifier combination
+`X-Rate-Limit-Remaining` - The number of requests left for the specific client/IP/device identifier combination during the current rate-limit window
+`X-Rate-Limit-Reset` - The time when the rate limit resets, specified in [UTC epoch time](https://www.epochconverter.com/) for the specific client/IP/device identifier combination
+
+For example:
+
+```
+HTTP/1.1 200 OK
+X-Rate-Limit-Limit: 60
+X-Rate-Limit-Remaining: 35
+X-Rate-Limit-Reset: 1516307596
+```
+
+When a specific client/IP/device identifier combination exceeds either the per minute request limit (500 request per minute) or the concurrent limit (two concurrent requests), the `/authorize` request returns an HTTP 429 error.
+
+### How to enable this feature
```

Comment:
This will change based on @ZoeMonosson-okta's final mocks

Reply:
Admin console->settings->scroll down to client baseed rate limit (section)

@ZoeMonosson-okta  can you add the mock of the settings page here?


![](https://files.slack.com/files-pri/T6WPNMPFU-F01BU2LE58U/client_based_rate_limiting_form-current.png)

Reply:
Client based should be hyphenated since it is describing rate limiting: Client-based rate limiting

Reply:
Thanks Susan, Michael T will update the UI with hyphen `client-based`. I don't think a period after "system log" makes sense since we have a new tab icon